consensus/src/round_manager.rs

```diff
@@ -1023,6 +1120,41 @@ impl RoundManager {
         }
     }
 
+    async fn process_order_vote_reception_result(
+        &mut self,
+        order_vote: &OrderVote,
+        result: OrderVoteReceptionResult,
+    ) -> anyhow::Result<()> {
+        match result {
+            OrderVoteReceptionResult::NewLedgerInfoWithSignatures(ledger_info_with_signatures) => {
+                if let Some(order_voted_block) = self
+                    .block_store
+                    .get_block(order_vote.ledger_info().consensus_block_id())
+                {
+                    if !order_voted_block.block().is_nil_block() {
+                        observe_block(
+                            order_voted_block.block().timestamp_usecs(),
+                            BlockStage::QC_AGGREGATED,
```

Comment:
this is not QCAggregated, we need a different label for this

Reply:
Done.